sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs

```diff
@@ -1123,7 +1124,7 @@ public static partial class ArmAvsModelFactory
+        public static Azure.ResourceManager.Avs.AvsPrivateCloudData AvsPrivateCloudData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, System.Collections.Generic.IDictionary<string, string> tags = null, Azure.Core.AzureLocation location = default(Azure.Core.AzureLocation), string skuName = null, Azure.ResourceManager.Models.ManagedServiceIdentity identity = null, Azure.ResourceManager.Avs.Models.AvsManagementCluster managementCluster = null, Azure.ResourceManager.Avs.Models.InternetConnectivityState? internet = default(Azure.ResourceManager.Avs.Models.InternetConnectivityState?), System.Collections.Generic.IEnumerable<Azure.ResourceManager.Avs.Models.SingleSignOnIdentitySource> identitySources = null, Azure.ResourceManager.Avs.Models.PrivateCloudAvailabilityProperties availability = null, Azure.ResourceManager.Avs.Models.CustomerManagedEncryption encryption = null, System.Collections.Generic.IEnumerable<string> extendedNetworkBlocks = null, Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState? provisioningState = default(Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState?), Azure.ResourceManager.Avs.Models.ExpressRouteCircuit circuit = null, Azure.ResourceManager.Avs.Models.AvsPrivateCloudEndpoints endpoints = null, string networkBlock = null, string managementNetwork = null, string provisioningNetwork = null, string vMotionNetwork = null, string vCenterPassword = null, string nsxtPassword = null, string vCenterCertificateThumbprint = null, string nsxtCertificateThumbprint = null, System.Collections.Generic.IEnumerable<Azure.Core.ResourceIdentifier> externalCloudLinks = null, Azure.ResourceManager.Avs.Models.ExpressRouteCircuit secondaryCircuit = null, Azure.ResourceManager.Avs.Models.NsxPublicIPQuotaRaisedEnum? nsxPublicIPQuotaRaised = default(Azure.ResourceManager.Avs.Models.NsxPublicIPQuotaRaisedEnum?)) { throw null; }
```

Comment:
This adds `System.Collections.Generic.IEnumerable<string> extendedNetworkBlocks = null,`, not sure my CI is complaining about API compatibility. https://github.com/Azure/azure-sdk-for-net/pull/39536/checks?check_run_id=19520494032

Reply:
> Error : MembersMustExist : Member 'public Azure.ResourceManager.Avs.AvsPrivateCloudData Azure.ResourceManager.Avs.Models.ArmAvsModelFactory.AvsPrivateCloudData(Azure.Core.ResourceIdentifier, System.String, Azure.Core.ResourceType, Azure.ResourceManager.Models.SystemData, System.Collections.Generic.IDictionary<System.String, System.String>, Azure.Core.AzureLocation, System.String, Azure.ResourceManager.Models.ManagedServiceIdentity, Azure.ResourceManager.Avs.Models.AvsManagementCluster, System.Nullable<Azure.ResourceManager.Avs.Models.InternetConnectivityState>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.Avs.Models.SingleSignOnIdentitySource>, Azure.ResourceManager.Avs.Models.PrivateCloudAvailabilityProperties, Azure.ResourceManager.Avs.Models.CustomerManagedEncryption, System.Nullable<Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState>, Azure.ResourceManager.Avs.Models.ExpressRouteCircuit, Azure.ResourceManager.Avs.Models.AvsPrivateCloudEndpoints, System.String, System.String, System.String, System.String, System.String, System.String, System.String, System.String, System.Collections.Generic.IEnumerable<Azure.Core.ResourceIdentifier>, Azure.ResourceManager.Avs.Models.ExpressRouteCircuit, System.Nullable<Azure.ResourceManager.Avs.Models.NsxPublicIPQuotaRaisedEnum>)' does not exist in the implementation but it does exist in the contract.

Reply:
Was able to add the old `AvsPrivateCloudData  ` method to `customized/ArmAvsModelFactory.cs public static partial class ArmAvsModelFactor`